lib/tasks/tribunal_case.rake

```diff
@@ -0,0 +1,10 @@
+namespace :tribunal_case do
```

Comment:
Can we have this in the daily_tasks.rake file? It will make it easier to see what's going on. Could you also add an  "invoke" line in the main daily_tasks rake task?